base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -428,17 +428,18 @@ impl ConsensusConstants {
     pub fn igor() -> Vec<Self> {
         let mut algos = HashMap::new();
         // sha3/monero to 40/60 split
```

Comment:
```suggestion
        // sha3/monero to 25/75 split
```
thats the correct split here
avg block time = (10+15) / 2 = 7.5
thus percentage monero block = 7.5/x = 10, then 75%. 
Thus sha3 = 25%

Formula for calculating the numbers here is specified in `struct PowAlgorthimConstants`
Target time = {avg block time}/{Desried block percentage} = {algo target time}

Reply:
Perhaps we should change this to make it correct 40/60

Reply:
It should be 40/60

Reply:
Updated as:
```rust
        // For SHA3/Monero to have a 40/60 split:
        // sha3_target_time = monero_target_time * (100 - 40) / 40
        // monero_target_time = sha3_target_time * (100 - 60) / 60
        // target_time = monero_target_time * sha3_target_time / (monero_target_time + sha3_target_time)
        const MONERO_SPLIT: u64 = 60;                             // Must be < 100
        const SHA3_SPLIT: u64 = 100 - MONERO_SPLIT;
        const MONERO_TARGET_TIME: u64 = 20;                       // usually 200 (for a 120 average block time)
        const SHA3_TARGET_TIME: u64 = MONERO_TARGET_TIME * (100 - SHA3_SPLIT) / SHA3_SPLIT;
        algos.insert(PowAlgorithm::Sha3, PowAlgorithmConstants {
            max_target_time: SHA3_TARGET_TIME * 6,                // target_time x 6
            min_difficulty: (SHA3_TARGET_TIME * 200_000).into(),  // target_time x 200_000
            max_difficulty: u64::MAX.into(),
            target_time: SHA3_TARGET_TIME,                        // (monero_target_time x 1.5)
        });
        algos.insert(PowAlgorithm::Monero, PowAlgorithmConstants {
            max_target_time: MONERO_TARGET_TIME * 6,           // target_time x 6
            min_difficulty: (MONERO_TARGET_TIME * 300).into(), // target_time x 300
            max_difficulty: u64::MAX.into(),
            target_time: MONERO_TARGET_TIME,
        });
```